lib/bulk_data_test_kit/v1.0.1/bulk_data_status_check_test.rb

```diff
@@ -22,6 +22,24 @@ class BulkDataStatusCheckTest < Inferno::Test
       DESCRIPTION
       # link 'http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request'
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@123',
+                            'hl7.fhir.uv.bulkdata_2.0.0@125',
+                            'hl7.fhir.uv.bulkdata_2.0.0@132',
+                            'hl7.fhir.uv.bulkdata_2.0.0@144',
+                            'hl7.fhir.uv.bulkdata_2.0.0@145',
+                            'hl7.fhir.uv.bulkdata_2.0.0@147',
+                            'hl7.fhir.uv.bulkdata_2.0.0@150',
+                            'hl7.fhir.uv.bulkdata_2.0.0@153',
+                            'hl7.fhir.uv.bulkdata_2.0.0@154',
```

Comment:
I was reading 154 and was thinking that it would be covered if the transaction time was after the modified time, and i think it would always be? I can remove the reference to 154 if that is not correct.